packages/database-model/src/sequelize-model.js

```diff
@@ -96,6 +96,28 @@ async function defineSites (sequelize) {
   })
 }
 
+async function defineEcologistExperience (sequelize) {
+  models.ecologistExperience = await sequelize.define('ecologist-experience', {
+    id: { type: DataTypes.UUID },
+    experience: { type: DataTypes.JSONB },
+    applicationId: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      references: {
+        model: models.applications,
+        key: 'id'
+      }
+    },
+    submitted: { type: DataTypes.DATE },
```

Comment:
Probably not necessary as the experience will be submitted at the same time as the application, but leave it for now.